File: Python/Level0/문자열 반복 출력.py
# input 함수를 통해 사용자에게 값을 요청 후 변수 a, b에 값을 담는다
a, b = input("반복할 문자와 횟수를 입력해주세요.").strip().split(' ')
b = int(b)

# a라는 문자를 b번 반복
print(a * b)

File: JavaScript/Level0/문자열반복출력.js
const readline = require("readline");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
  input = line.split(" ");
}).on("close", function () {
  str = input[0];
  n = Number(input[1]);

  // str에 담긴 문자열를 n번 반복 출력
  console.log(str.repeat(n));
});
